packages/hooks/package.json

```diff
@@ -17,7 +17,6 @@
   },
   "dependencies": {
     "@leafygreen-ui/lib": "workspace:^",
-    "@leafygreen-ui/palette": "workspace:^",
```

Comment:
This was added when `useResizable` was included with `hooks`, but should be removed since `useResizable` is in its own package